src/components/shop/shop-menu-screen.js

```diff
@@ -26,28 +26,24 @@ export class ShopMenu extends Screen {
         this.transientData.shop = { config: this.config.shopConfig };
         setBalance(this);
 
-        this.menu = createMenu(this);
+        const resize = createMenu(this);
```

Comment:
I don't like either way of doing this, it reads strange. Not sure how better to do it though..

Reply:
Yeah - might talk about that in the dev channel - it actually seems to happen a lot - twice in this project.

I agree - feels dirty.